roles/backend_setup/tasks/vdo_create.yml

```diff
@@ -1,4 +1,16 @@
 ---
+- name: Set pyyaml package name
+  set_fact:
+    pyyaml_name: python-yaml
+- name: Overwrite pyyaml package name for non-Debian
+  set_fact:
+    pyyaml_name: PyYAML
+  when: ansible_distribution != "Debian" and ansible_distribution != "Ubuntu"
+- name: Install PyYAML for non-debian
+  package:
+    name: "{{ pyyaml_name }}"
+    state: present
```

Comment:
Same here, you can put a check to install this only on Debian/Ubuntu.

Reply:
Sorry for missing this earlier. python-yaml installation under VDO creation is not needed. This package is installed as part of ansible installation. I think it is the same with deb.